src/textual/widgets/_tree.py

```diff
@@ -473,8 +473,9 @@ def __init__(
         name: str | None = None,
         id: str | None = None,
         classes: str | None = None,
+        disabled: bool = False,
```

Comment:
Ditto here about the docstring.
(We may end up realising it suffices to document once in `Widget`. I don't know by heart.)

Reply:
> We may end up realising it suffices to document once in Widget. 

Yeah, that's a tricky one isn't it? Often the parameters can just be "inherited", as it were, and I've see autodoc systems that take that into account. On the other hand some of the widgets "inherit" almost all of the parameters but add one or two of their own, which suggests needing to repeat everything. :-/

Reply:
Oh, and yeah, `Tree` seems to be lacking `__init__` documentation on the whole, so I'll leave that to a followup PR as I don't really like turning a focused PR into a more general one if I can help it.